scripts/build

```diff
@@ -165,10 +165,16 @@ if [ ! -f $STAMP ]; then
     . $PKG_DIR/package.mk
 
     if [ "$TARGET" = "target" ]; then
+      if [ "$PKG_IS_PYTHON" = "yes" ]; then
+        PKG_DEPENDS_TARGET="toolchain distutilscross:host Python $PKG_DEPENDS_TARGET"
+      fi
       for p in $PKG_DEPENDS_TARGET; do
         $SCRIPTS/build $p
       done
     elif [ "$TARGET" = "host" ]; then
+      if [ "$PKG_IS_PYTHON" = "yes" ]; then
+        PKG_DEPENDS_HOST="toolchain distutilscross:host $PKG_DEPENDS_TARGET"
```

Comment:
If you move that outside of the if "$TARGET" = "host" or "target" you can save one if-statement. Just check:
`if [ "$PKG_IS_PYTHON" = "yes" ]; then
    PKG_DEPENDS_TARGET="toolchain distutilscross:host Python $PKG_DEPENDS_TARGET";
    PKG_DEPENDS_HOST="toolchain distutilscross:host $PKG_DEPENDS_TARGET";
fi`